packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -93,21 +91,37 @@ export const Avatar = (props: AvatarProps) => {
 
 function InitialsAvatarFallback(props: AvatarProps) {
   const initials = getInitials(props);
+  const { parsedInternalTheme } = useAppearance();
 
   return (
-    <Text
-      as='span'
-      sx={{ ...common.centeredFlex('inline-flex'), width: '100%' }}
+    <Flex
+      sx={{ position: 'relative' }}
+      justify='center'
+      align='center'
     >
-      {initials}
-    </Text>
+      <BoringAvatar
+        size={props.size?.(parsedInternalTheme)}
+        name={`${props.firstName} ${props.lastName}`}
+        colors={['#CEEBD1', '#B6DEB9', '#B1CCB4', '#AEBFAF', '#A6ADA7']}
```

Comment:
⚠️ These values should be coming from the theme, otherwise, they won't be customizable by the end user.
Let's discuss this because its possible that we'd need to provide `variables.colorAvatar` or similar API for that.

Reply:
We hardcoded them on purpose, for now, to test what it looks like. We will get back to this.